tensorboardX/torchvis.py

```diff
@@ -26,7 +26,7 @@ def register(self, *args):
         formats = ['tensorboard'] if not args else args
         for format in formats:
             if self.subscribers.get(format) is None and format in vis_formats.keys():
-                self.subscribers[format] = vis_formats[format]()
+                self.subscribers[format] = vis_formats[format](**init_kwargs[format])
```

Comment:
init_kwargs should be passed to register()

Reply:
Done. Also changed `**init_kwargs[format]` to `**init_kwargs.get(format, {})`, to avoid failure if no specification is given